### ISSUE TITLE
Got a Crash

### ISSUE THREAD

**Issue by nitinmishra27** [xizzhu/simple-tool-tip#3]
```
ToolTip toolTip = new ToolTip.Builder()
                    .withText("Simple Tool Tip!")
                    .withBackgroundColor(Color.WHITE)
                    .withCornerRadius(5)
                    .withPadding(10, 10, 10, 10)
                    .withTextSize(18)
                    .withTextColor(Color.BLACK)
                    .build();
            ToolTipView toolTipView = new ToolTipView.Builder(getActivity())
                    .withAnchor(view)
                    .withToolTip(toolTip)
                    .withGravity(Gravity.START)
                    .build();
            toolTipView.show();
```

java.lang.NullPointerException: Attempt to write to field 'int android.view.ViewGroup$MarginLayoutParams.topMargin' on a null object reference
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at com.github.xizzhu.simpletooltip.ToolTipView.onPreDraw(ToolTipView.java:286)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.view.ViewTreeObserver.dispatchOnPreDraw(ViewTreeObserver.java:944)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2000)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1084)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5990)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.view.Choreographer$CallbackRecord.run(Choreographer.java:767)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.view.Choreographer.doCallbacks(Choreographer.java:580)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.view.Choreographer.doFrame(Choreographer.java:550)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:753)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.os.Handler.handleCallback(Handler.java:746)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.os.Handler.dispatchMessage(Handler.java:95)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.os.Looper.loop(Looper.java:135)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at android.app.ActivityThread.main(ActivityThread.java:5343)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at java.lang.reflect.Method.invoke(Native Method)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at java.lang.reflect.Method.invoke(Method.java:372)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:907)
10-21 10:44:08.272 5376-5376/com.sfwtech.hoddoctor E/ACRA:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:702)

**Comment by xizzhu** [comment]
`Gravity.START` is not supported yet, I'll add that soon.

**Comment by xizzhu** [comment]
Fixed in 81914d9d8810e07fe15b07e64d4704f2a03c5504
